src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
> I think last highest RID would meet my needs.

To be precise, this is only for Windows and OSX - Linux will fall to linux-*. @Petermarcu are you fine with this as well? 

> Maybe we stop using osx.10.10 in our packages as the "base OSX" RID. And if this asset will work on any OSX that we support, we start putting that asset under plainly osx. Same goes for Windows. Don't put assets under win7. Instead put them in win.

I think that is the plan.